dev/tests/integration/phpunit.xml.dist

```diff
@@ -19,7 +19,6 @@
         <testsuite name="Magento Integration Tests">
             <directory suffix="Test.php">testsuite</directory>
             <directory suffix="Test.php">../../../update/dev/tests/integration/testsuite</directory>
-            <exclude>testsuite/Magento/Test/Integrity</exclude>
```

Comment:
In nature, some are very similar to static tests, but happen to rely on certain parts of the application being available as a source for the files they are running tests on. Others are literally testing to make sure the application _generated_ files are in working order. So they are a sort of integrated static test if you will. I get that. So perhaps they do belong broken out into their own testing domain, but doing that properly would look something like having them split into a separate _testsuite_ in terms of phpunit configuration. This would not in my opinion necessitate running them in a separate job.

Is it important that they run separately? I doubt it, and that's why I wrapped them back into the primary integration builds. It doesn't make sense to parallelize this portion of the integration tests, particularly since these tests take about 60 seconds to run, but the job in total takes about 5-6 minutes because of the time it takes to stand up a worker to run them.

The goal I'm trying to achieve, and why I started messing with this, is making the Travis builds spend far less time running, making the experience of those submitting PRs and testing code they push up to their own forks (may it be public or private) better. Pulling all the integration tests together cuts another 4-5 minutes off the build process. :)